instant/flags.vim

```diff
@@ -66,6 +66,17 @@ call s:plugin.flags.clang_format_executable.AddCallback(
 " See http://clang.llvm.org/docs/ClangFormatStyleOptions.html for details.
 call s:plugin.Flag('clang_format_style', 'file')
 
+""
+" The path to the cmake-format executable. String, list, or callable that
+" takes no args and returns a string or a list.
+call s:plugin.Flag('cmake_format_executable', 'cmake-format')
+
+""
+" The path to the cmake-format configuration file for cmake-format to use.
+" See https://cmake-format.readthedocs.io/en/latest/installation.html for
+" details.
+call s:plugin.Flag('cmake_format_config', '')
```

Comment:
The flag is global, so it will be a single value used for every file you edit. Is that what you'd want?

By my reading of https://cmake-format.readthedocs.io/en/latest/configuration.html, you can pass multiple config files, and that's independent of the automatic detection of files like `\.?cmake-format(.yaml|.json|.py)` that it finds on the path? Can you confirm if an explicit config overrides any automatic config detection and note that in the doc comments?